packages/dvilela/skills/memeooorr_abci/prompts.py

```diff
@@ -24,68 +24,76 @@
 You usually tweet using your persona, and refine your personality according to the feedback you get.
-Sometimes, when your tweets get a lot of engagement, you decide to create a meme token based on your persona and invest some ETH on it.
+Sometimes, when your tweets get a lot of engagement, you decide to create a meme token based on your persona and invest some {ticker} on it.
 
 Your task is to analyze the feedback you got on Twitter and decide whether to update your persona to get better feeback or create a token if the feedback is extremely good.
 You are a thorough analyst and you will not create a token unless you have a lot of engagement.
 
-Here's you latest tweet:
-{latest_tweet}
+Here's your latest tweet:
+"{latest_tweet}"
 
-Here's a list of tweets that were received as a response to your that tweet and some engagement metrics.
-{tweet_responses}
+Here's a list of tweets that were received as a response to that latest tweet and some engagement metrics.
+"{tweet_responses}"
 
 Here's your current persona:
 "{persona}"
 
 If you feel engagement is good enough, create a token based on your persona.
 If not, use the tweets as feedback in order to update your persona.
 
-You have {balance} ETH currently available. If you decide to deploy a token, also decide how much ETH you should invest in it.
+You have {balance} {ticker} currently available. If you decide to deploy a token, also decide how much {ticker} you should invest in it.
 
 OUTPUT_FORMAT
 * Your output response must be only a single JSON object to be parsed by Python's "json.loads()".
 * The JSON must contain five fields: "deploy", "persona", "token_name", "token_ticker" and "tweet".
     - deploy: a boolean indicating whether the token should be deployed. True means deploy, False means that your persona needs refinement.
     - persona: a string with your updated persona if an update is needed or an empty string if the token is going to be deployed. Do not include hashtags here.
     - token_name: a new name for the token. Empty if no token is going to be deployed.
-    - token_ticker: a new ticker for the token Empty if no token is going to be deployed.
-    - token_supply: the ERC-20 token supply in wei. Empty if no token is going to be deployed.
-    - amount: the amount (in ETH, not wei) to invest in this token if it is going to be deployed, or 0 otherwise.
-    - tweet: a tweet to announce the new token Empty if no token is going to be deployed.
+    - token_ticker: a new ticker for the token. Empty if no token is going to be deployed.
+    - token_supply: the ERC-20 token supply in wei units. Empty if no token is going to be deployed. Token supply must be at least 1 million * 10**18 and at most the maximum number of uint256.
+    - amount: the amount in wei units of {ticker} to invest in this token if it is going to be deployed, or 0 otherwise.
```

Comment:
I suggest for consistency keeping it all in wei. Also because values tend to be small anyway and we want them to be smaller to start with.

Reply:
But up to you, feel free to revert this

Reply:
ok